Bluepill-cli/Bluepill-cli/Simulator/SimulatorRunner.m

```diff
@@ -190,6 +217,43 @@ + (void)createDeviceWithConfig:(BPConfiguration *)config andName:(NSString *)dev
                        }];
 }
 
++ (SimDevice *)findDeviceWithConfig:(BPConfiguration *)config andDeviceID:(NSUUID *)deviceID {
```

Comment:
Nah, by passing a NSError back to the caller, it is up to the caller to decide how to deal with the error (suppress, error handling etc), instead of printing things out just here.